tests/resources/trustyai/trustyai_crd.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: trustyai.opendatahub.io.trustyai.opendatahub.io/v1alpha1
+kind: TrustyAIService
+metadata:
+  name: trustyai-service
+spec:
+  # Optional values for replicas, image and tag. Below are the default values.
```

Comment:
I think these comments can be deleted, as they are documented elsewhere. wdyt?

Reply:
done